src/components/Checkout.js

```diff
@@ -60,7 +60,24 @@ class Checkout extends React.Component {
         </div>
       );
     } else {
```

Comment:
the linter didnt grab this? the `else` is unnecessary if you just return something in the else block